pandas/io/formats/html.py

```diff
@@ -320,9 +320,15 @@ def _write_header(self, indent):
 
         self.write('</thead>', indent)
 
+    def _get_formatted_values(self):
+        with option_context('display.max_colwidth', 999999):
```

Comment:
Can just use `None` here

Reply:
would seem logical, but raises:

`ValueError: Value must have type '<class 'int'>'`

there is a bug/feature where values less than 4 are ignored, see #16097 but that would be more of a hack than using 999999, though would work with strings of length greater than 999999 chars 😀